app/models/spotlight/featured_image.rb

```diff
@@ -4,16 +4,22 @@ module Spotlight
   class FeaturedImage < ActiveRecord::Base
     mount_uploader :image, Spotlight::FeaturedImageUploader
 
-    before_validation :set_image_from_uploaded_resource
+    # before_validation :set_image_from_uploaded_resource
```

Comment:
I believe this was only necessary under carrierwave.  Things seem to work without it.  I'll remove this line as well as the method below.